api/serializers.py

```diff
@@ -38,3 +38,9 @@ class VideoSerializer(serializers.ModelSerializer):
     class Meta:
         model = Video
         fields = ("id", "title")
+
+
+class ReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Report
+        fields = ("report_type", "reporter", "reportee", "description")
```

Comment:
As of right now, I just don't know what data will be wanted on the frontend so I just left it as these. I didn't want to use `__all__` because I favor being explicit in the serializer in case fields are added to the model that we don't want to serialize.